components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -309,12 +307,13 @@ func CheckAndMountCACertBundle(ctx context.Context, cli client.Client, notebook
 				},
 			}
 			volumeMounts = append(volumeMounts, customCAVolumeMounts...)
+			log.Info("Certificates in 'ca-bundle.crt' are valid and can be mounted")
 		} else {
 			log.Error(err, "Error validating certificates for ca-bundle.crt")
 		}
 
 		if err := certValidator(cm, "odh-ca-bundle.crt", log); err == nil {
-			log.Info("Validating certificates for odh-ca-bundle.crt")
+			log.Info("Validating certificates for odh-ca-bundle.crt. They can not be mounted")
```

Comment:
I don't get this message TBH :thinking: At this line the certificates are validated successfully already, correct?

Reply:
ohh, you are right! my bad